app/SignInWithGithub/SignInWithGithub.tsx

```diff
@@ -0,0 +1,99 @@
+"use client"
+import { signInWithPopup,GithubAuthProvider,signOut } from 'firebase/auth'
+import React, { useEffect, useState } from 'react'
+import {auth,provider} from '../../lib/firebase-config'
+import { useRouter } from 'next/router'
+import Image from 'next/image'
+import { toast,ToastContainer } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css';
+
+const SignInWithGithub=()=>{
+    const router = useRouter();
+    const [imageURL, setImageURL] = useState<string | null>(null);
+    const [authenticated,setAuthenticated] = useState(false);
+    const signIn = async () => {
+        try {
+            const result = await signInWithPopup(auth, provider);
+            const credential = GithubAuthProvider.credentialFromResult(result);
+            console.log("Credential",credential);
+            console.log("Result: ",result);
+            const token: any = result.user.accessToken;
+            const username:any = result.user.displayName;
+            localStorage.setItem('accessToken', token);
+            const currDate = new Date().getTime;
+            document.cookie = `accessToken=${token};path=/; expires=${currDate}`;
+            const imgURL:any = result.user.photoURL;
+            localStorage.setItem('imageURL',imgURL);
```

Comment:
@Anmol-Baranwal, on this line, my terminal states this:
"Unexpected any. Specify a different type". Maybe this plays a role in the deployment failure?